modules/imgproc/src/subdivision2d.cpp

```diff
@@ -852,6 +853,11 @@ void Subdiv2D::checkSubdiv() const
     }
 }
 
+bool Subdiv2D::isWithinRect(Point2f pt) const
+{
+    return (topLeft.x <= pt.x && pt.x <= bottomRight.x) && (topLeft.y <= pt.y && pt.y <= bottomRight.y);
```

Comment:
Try to reuse existed functionality: `Point2f::inside()` or `Rect::contains()`

Note: right/bottom side of OpenCV rectangles is "open" (exclusive).